diff -Naur a/NickvisionTubeConverter.Shared/Linux/org.nickvision.tubeconverter.desktop.in b/NickvisionTubeConverter.Shared/Linux/org.nickvision.tubeconverter.desktop.in
--- a/NickvisionTubeConverter.Shared/Linux/org.nickvision.tubeconverter.desktop.in	2023-06-13 00:34:09.996218531 +0530
+++ b/NickvisionTubeConverter.Shared/Linux/org.nickvision.tubeconverter.desktop.in	2023-06-13 00:42:47.352870719 +0530
@@ -3,7 +3,7 @@
 Name=Tube Converter
 Comment=Get video and audio from the web
 Exec=@EXEC@
-Icon=org.nickvision.tubeconverter
+Icon=snap.tube-converter.org.nickvision.tubeconverter
 Terminal=false
 Type=Application
 Categories=AudioVideo;Network;
diff -Naur a/NickvisionTubeConverter.Shared/Linux/org.nickvision.tubeconverter.metainfo.xml.in b/NickvisionTubeConverter.Shared/Linux/org.nickvision.tubeconverter.metainfo.xml.in
--- a/NickvisionTubeConverter.Shared/Linux/org.nickvision.tubeconverter.metainfo.xml.in	2023-06-11 21:08:29.643511112 +0530
+++ b/NickvisionTubeConverter.Shared/Linux/org.nickvision.tubeconverter.metainfo.xml.in	2023-06-13 00:39:01.039920329 +0530
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <component type="desktop-application">
   <id>org.nickvision.tubeconverter</id>
+  <icon type="stock">snap.tube-converter.org.nickvision.tubeconverter</icon>
   <metadata_license>CC0-1.0</metadata_license>
   <project_license>MIT</project_license>
   <name>Tube Converter</name>
